bundles/org.openhab.core.ui.icon/src/test/java/org/openhab/core/ui/icon/internal/IconServletTest.java

```diff
@@ -66,7 +66,7 @@ public boolean isReady() {
         }
 
         public String getOutput() {
-            return new String(outputStream.toByteArray());
+            return outputStream.toString();
```

Comment:
Is this guaranteed to be the same?

Reply:
Yes it saves creating a copy of the byte array:

## ByteArrayOutputStream

```java
public synchronized byte[] toByteArray() {
    return Arrays.copyOf(buf, count);
}

public synchronized String toString() {
    return new String(buf, 0, count);
}
```

## String

```java
public String(byte[] bytes) {
    this(bytes, 0, bytes.length);
}
```